### PR TITLE
convert to actions

### PR DESCRIPTION
**Pull request by jason-jackson**
### Changed
- Convert to GitHub Actions

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -13,19 +13,17 @@ jobs:
...
+      - name: Set up Go
+        uses: actions/setup-go@v4
         with:
           go-version: 1.19
-      -
-        name: Run GoReleaser
-        uses: goreleaser/goreleaser-action@v2
+
+      - name: Run GoReleaser
+        uses: goreleaser/goreleaser-action@v5
```
[the remark this answers is not in the document]
Comment (briskt) [reply .github/workflows/release.yml:26]: In fact, it looks like [I gave up on v4](https://github.com/silinternational/tfc-ops/commit/db38eddb5400f8fda8a4378deb7db50108c065fc). Not that it couldn't be made to work, but just letting you know it's not a drop-in.

**Review by briskt — CHANGES_REQUESTED** [review]
The goreleaser change needs some work. Otherwise, I think it's fine.

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+version: "3"
```
Comment (briskt) [inline docker-compose.yml:1]: I think this file can vanish now.

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -1,10 +1,9 @@
 name: goreleaser
 
 on:
-  pull_request:
   push:
     tags:
-      - '*'
+      - "*"
```
Comment (briskt) [inline .github/workflows/release.yml:6]: What's wrong with single quotes?

**Review by briskt — APPROVED** [review]

File: .github/workflows/release.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (jason-jackson) [reply .github/workflows/release.yml:25]: Yeah, it's odd that it *fails* now that tag is missing, I decided to drop it from running on pull_request as we were only really interested in having it for releases, unless I'm mistaken.

File: .github/workflows/release.yml (hunk 3 above)
Comment (jason-jackson) [reply .github/workflows/release.yml:6]: I blame my auto-formatter. I should look into getting it to be more chill

File: .github/workflows/release.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (briskt) [reply .github/workflows/release.yml:25]: I _think_ that's fine. I can't remember why I had it on pull_request. Maybe just as a trial run.

[the remark this answers is not in the document]
Comment (jason-jackson) [reply .github/workflows/release.yml:25]: It's what the go-releaser action [says](https://github.com/goreleaser/goreleaser-action). Though they also mention you can do it without pull_request too.

**Pull request merged**